src/core/directives/google-map.ts

```diff
@@ -279,17 +308,13 @@ export class SebmGoogleMap implements OnChanges, OnInit, OnDestroy {
   }
 
   /* @internal */
-  ngOnChanges(changes: {[propName: string]: SimpleChange}) {
-    this._updateMapOptionsChanges(changes);
+  ngOnChanges(changes: SimpleChanges) {
+    this._updateMapOptionsChanges();
```

Comment:
This updates everything because there is no change tracking any more. Can you revert that?